diff --git a/node_modules/@sumsub/react-native-mobilesdk-module/android/build.gradle b/node_modules/@sumsub/react-native-mobilesdk-module/android/build.gradle
index 1234567..abcdefg 100644
--- a/node_modules/@sumsub/react-native-mobilesdk-module/android/build.gradle
+++ b/node_modules/@sumsub/react-native-mobilesdk-module/android/build.gradle
@@ -69,9 +69,9 @@ dependencies {
     implementation "com.sumsub.sns:idensic-mobile-sdk:1.40.2"
 
     // remove comment to enable Device Intelligence
-    // implementation "com.sumsub.sns:idensic-mobile-sdk-fisherman:1.40.2"
+    implementation "com.sumsub.sns:idensic-mobile-sdk-fisherman:1.40.2"
     // remove comment if you need VideoIdent support
-    // implementation "com.sumsub.sns:idensic-mobile-sdk-videoident:1.40.2"
+    implementation "com.sumsub.sns:idensic-mobile-sdk-videoident:1.40.2"
     // remove comment if you need EID support
