diff --git a/node_modules/react-native-gesture-handler/apple/RNGestureHandlerModule.mm b/node_modules/react-native-gesture-handler/apple/RNGestureHandlerModule.mm
index 0000002..0000003 100644
--- a/node_modules/react-native-gesture-handler/apple/RNGestureHandlerModule.mm
+++ b/node_modules/react-native-gesture-handler/apple/RNGestureHandlerModule.mm
@@ -99,7 +99,7 @@ void decorateRuntime(jsi::Runtime &runtime)
         if (!arguments[0].isObject()) {
           return jsi::Value::null();
         }
-        auto shadowNode = shadowNodeFromValue(runtime, arguments[0]);
+        auto shadowNode = arguments[0].asObject(runtime).getNativeState<ShadowNodeWrapper>(runtime)->shadowNode;
         bool isFormsStackingContext = shadowNode->getTraits().check(ShadowNodeTraits::FormsStackingContext);
         return jsi::Value(isFormsStackingContext);
       });
diff --git a/node_modules/react-native-gesture-handler/android/build.gradle b/node_modules/react-native-gesture-handler/android/build.gradle
index 0000000..0000001 100644
--- a/node_modules/react-native-gesture-handler/android/build.gradle
+++ b/node_modules/react-native-gesture-handler/android/build.gradle
@@ -202,9 +202,10 @@
 }
 
 def kotlin_version = safeExtGet('kotlinVersion', project.properties['RNGH_kotlinVersion'])
+def reactNativeDependency = safeExtGet("reactNativeDependency", "com.facebook.react:react-android")
 
 dependencies {
-    implementation 'com.facebook.react:react-native:+' // from node_modules
+    implementation reactNativeDependency
     
 
     if (shouldUseCommonInterfaceFromReanimated()) {
