diff --git a/android/src/main/java/com/airbnb/android/react/lottie/LottieAnimationViewPropertyManager.kt b/android/src/main/java/com/airbnb/android/react/lottie/LottieAnimationViewPropertyManager.kt
index 1b1bf37..89a3d6e 100644
--- a/android/src/main/java/com/airbnb/android/react/lottie/LottieAnimationViewPropertyManager.kt
+++ b/android/src/main/java/com/airbnb/android/react/lottie/LottieAnimationViewPropertyManager.kt
@@ -1,5 +1,6 @@
 package com.airbnb.android.react.lottie
 
+import android.graphics.Color
 import android.graphics.ColorFilter
 import android.graphics.Typeface
 import android.net.Uri
@@ -18,6 +19,5 @@ import com.facebook.react.bridge.ReadableMap
 import com.facebook.react.bridge.ReadableType
 import com.facebook.react.views.text.ReactFontManager
-import com.facebook.react.views.text.TextAttributeProps.UNSET
 import com.facebook.react.util.RNLog
 import java.lang.ref.WeakReference
 import java.util.regex.Pattern
@@ -35,6 +35,7 @@ import java.io.FileInputStream
 class LottieAnimationViewPropertyManager(view: LottieAnimationView) {
     private val viewWeakReference: WeakReference<LottieAnimationView>
     private val TAG = "lottie-react-native"
+    private val UNSET = -1
 
 
     /**
@@ -246,7 +247,7 @@ class LottieAnimationViewPropertyManager(view: LottieAnimationView) {
         view: LottieAnimationView
     ) {
         val color: Int = if (colorFilter.getType("color") == ReadableType.Map) {
-            ColorPropConverter.getColor(colorFilter.getMap("color"), view.context)
+            ColorPropConverter.getColor(colorFilter.getMap("color"), view.context) ?: Color.TRANSPARENT
         } else {
             colorFilter.getInt("color")
         }
